technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -105,32 +107,34 @@
   <macro name="container-contributors">
     <choose>
       <if variable="container-author">
-        <names variable="container-author" suffix=", ">
-          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="first">
-            <name-part name="family"/>
-          </name>
-          <et-al term="et-al"/>
-        </names>
-        <text macro="container-title" font-style="normal"/>
-        <text macro="secondary-container-contributors"/>
+        <group delimiter=", ">
```

Comment:
I'm a bit confused -- this shouldn't change anything. Where is the comma intended to go?

Reply:
This is to take care of the problem of a double comma if a contianer author is present and no secondary container contributor(s) (as in author anthologies)

Reply:
never mind, I misread the code. Makes sense.